examples/matchmaker/mo/mo.go

```diff
@@ -0,0 +1,222 @@
+package mo
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/GoogleCloudPlatform/open-match/examples/matchmaker/divs"
+	"github.com/GoogleCloudPlatform/open-match/internal/pb"
+	"github.com/gobs/pretty"
+	"github.com/gogo/protobuf/jsonpb"
+)
+
+var (
+	teamSize int
+	numTeams int
+	debug    bool
+	regions  []string
+	tiers    []string
+)
+
+type poolArray []*pb.PlayerPool
+type ftrArray []*pb.Filter
+type rosterArray []*pb.Roster
+
+/*
+//example of how to use this module
+func main() {
+	moChan := make(chan *pb.MatchObject)
+	doneChan := make(chan bool)
+	go GenerateMatchObjects(moChan, doneChan)
+	for {
+		select {
+		case mo := <-moChan:
+			pretty.PrettyPrint(mo)
+		case <-doneChan:
+			log.Println("all Done")
+			return
+		}
+	}
+}
+*/
+
+// GenerateMatchObjects
+func GenerateMatchObjects(moChan chan *pb.MatchObject) {
+	//func GenerateMatchObjects(moChan chan *pb.MatchObject, doneChan chan bool) {
```

Comment:
Please remove dead/commented code if it's not relevant.

Reply:
done